examples/loader-google-bigquery/src/data/google-bigquery.js

```diff
@@ -0,0 +1,20 @@
+import "dotenv/config";
+import {BigQuery} from "@google-cloud/bigquery";
+
+const {BQ_PROJECT_ID, BQ_CLIENT_EMAIL, BQ_PRIVATE_KEY} = process.env;
+
+if (!BQ_PROJECT_ID) throw new Error(".env missing BQ_PROJECT_ID");
+if (!BQ_CLIENT_EMAIL) throw new Error(".env missing BQ_CLIENT_EMAIL");
+if (!BQ_PRIVATE_KEY) throw new Error(".env missing BQ_PRIVATE_KEY");
```

Comment:
```suggestion
if (!BQ_PROJECT_ID) throw new Error("missing BQ_PROJECT_ID");
if (!BQ_CLIENT_EMAIL) throw new Error("missing BQ_CLIENT_EMAIL");
if (!BQ_PRIVATE_KEY) throw new Error("missing BQ_PRIVATE_KEY");
```

These environment variables don’t _have_ to be defined in the `.env` (and typically won’t in a CI environment — the `.env` is instead of local development).

Reply:
my mistake